### ISSUE TITLE
Clavis Aethiopica

### ISSUE THREAD

**Issue by DariaElagina** [BetaMasaheft/Documentation#2560]
Dear Bm-Team,

is there any specific way to refer to Clavis Aethiopica in a bibliography? I could not find an entry for it in Zotero, should we create one?

Very best

Daria

**Comment by DariaElagina** [comment]
Thank you, @eu-genia , but if I am not mistaken he does not cite it in References, only in the footnotes.

**Comment by eu-genia** [comment]
Right, but he explains it. Compare also https://flore.unifi.it/retrieve/d0913e66-4bba-4c61-82d5-19d14fcb9aa6/Brita%202024_Matter%20and%20Devotion.pdf where CAe is explained in abbreviations.
